rs-matter/src/transport/session.rs

```diff
@@ -163,17 +168,11 @@ impl Session {
         self.peer_nodeid
     }
 
-    pub fn get_peer_cat_ids(&self) -> Option<&NocCatIds> {
+    pub fn get_local_fabric_idx(&self) -> u8 {
         match &self.mode {
-            SessionMode::Case(a) => Some(&a.cat_ids),
-            _ => None,
-        }
-    }
-
-    pub fn get_local_fabric_idx(&self) -> Option<u8> {
-        match &self.mode {
-            SessionMode::Case(a) => Some(a.fab_idx),
-            _ => None,
+            SessionMode::Case { fab_idx, .. } => fab_idx.get(),
+            SessionMode::Pase { fab_idx, .. } => *fab_idx,
+            _ => 0,
```

Comment:
nit, same as `SessionMode::fab_idx()`?

Reply:
You are right! Will replace with just `self.mode.fab_idx()` though we can also get rid of the getter.